ambry-account/src/main/java/com/github/ambry/account/AbstractAccountService.java

```diff
@@ -186,8 +189,13 @@ public Account getAccountById(short id) {
               if (existingContainer.isSameContainer(container)) {
                 // If an exactly same container already exists, treat as no-op (may be retry after partial failure).
                 // But include it in the list returned to caller to provide the containerId.
-                logger.info("Request to create container with existing name and properties: {}",
-                    existingContainer.toJson().toString());
+                String containerStr;
+                try {
+                  containerStr = objectMapper.writeValueAsString(existingContainer);
+                } catch (IOException e) {
+                  containerStr = existingContainer.toString();
```

Comment:
This basically means that processing the json has failed, meaning that there is something wrong with the container object. Should we bother trying to make progress after this? Maybe try throwing AccountServiceException here?

Reply:
serialize a container to a json in string usually doesn't fail. If it does, let's just print out the container name and id in the log message. This is what this code does. I don't think we should throw out an exception since we are serializing container just for log messages. I would rather just use container.toString() all the time if not for backward compatibility.

Reply:
Got it. Didn't realize that this was just for log message.